arch/arm/soc/atmel_sam/sam4s/soc.c

```diff
@@ -93,32 +52,34 @@ static ALWAYS_INLINE void clock_init(void)
 	/* Start the external crystal oscillator. */
 	PMC->CKGR_MOR = CKGR_MOR_KEY_PASSWD
 			/* Fast RC oscillator frequency is at 4 MHz. */
-			| CKGR_MOR_MOSCRCF_4MHz
+			| CKGR_MOR_MOSCRCF_4_MHz
 			/*
 			 * We select maximum setup time. While start up time
 			 * could be shortened this optimization is not deemed
 			 * critical right now.
 			 */
 			| CKGR_MOR_MOSCXTST(0xFFu)
 			/* RC OSC must stay on. */
-			| PMC_CKGR_MOR_MOSCRCEN
-			| PMC_CKGR_MOR_MOSCXTEN;
+			| CKGR_MOR_MOSCRCEN
+			| CKGR_MOR_MOSCXTEN;
 
 	/* Wait for oscillator to be stabilized. */
-	while (!(PMC->PMC_SR & PMC_SR_MOSCXTS))
+	while (!(PMC->PMC_SR & PMC_SR_MOSCXTS)) {
 		;
+	}
 
 	/* Select the external crystal oscillator as the main clock source. */
-	PMC->CKGR_MOR = PMC_CKGR_MOR_KEY
-			| PMC_CKGR_MOR_MOSCRCF_4MHZ
-			| PMC_CKGR_MOR_MOSCRCEN
-			| PMC_CKGR_MOR_MOSCXTEN
-			| PMC_CKGR_MOR_MOSCXTST
-			| PMC_CKGR_MOR_MOSCSEL;
+	PMC->CKGR_MOR = CKGR_MOR_KEY_PASSWD
+			| CKGR_MOR_MOSCRCF_4_MHz
+			| CKGR_MOR_MOSCRCEN
+			| CKGR_MOR_MOSCXTEN
+			| CKGR_MOR_MOSCXTST(0xFFu)
+			| CKGR_MOR_MOSCSEL;
 
 	/* Wait for external oscillator to be selected. */
-	while (!(PMC->PMC_SR & PMC_INT_MOSCSELS))
+	while (!(PMC->PMC_SR & PMC_SR_MOSCSELS)) {
 		;
+	}
 
 	/* Turn off RC OSC, not used any longer, to save power */
```

Comment:
Let's use one name, here we have "RC OSC" on line 84 we call it "RC oscillator". This may be confusing to a reader. "RC oscillator" is probably better.